react/ProductSummaryList.tsx

```diff
@@ -143,17 +143,19 @@ function ProductSummaryList(props: PropsWithChildren<Props>) {
   })
 
   const { products } = data ?? {}
+  // Not using ?? operator because listName can be ''
+  // eslint-disable-next-line no-unneeded-ternary
+  const listName = rawListName ? rawListName : 'List of products'
```

Comment:
You can use OR operator (`||`) for this.

Reply:
I think ternary is clearer in this case